curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-079.md

```diff
@@ -146,6 +169,8 @@ h1, h2 {
   width: 25%
 }
 
+/* FOOTER */
+
 --fcc-editable-region--
 
 --fcc-editable-region--
```

Comment:
```suggestion
--fcc-editable-region--

```
This helps prevent this:
![image](https://user-images.githubusercontent.com/51722130/121773596-d67b9400-cb74-11eb-93da-0f75c86abb0e.png)

Reply:
This is a feature, not a bug. 😉